streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -1082,7 +1081,7 @@ public void cleanUp() {
      * This will use the default Kafka Streams partitioner to locate the partition.
      * If a {@link StreamPartitioner custom partitioner} has been
      * {@link ProducerConfig#PARTITIONER_CLASS_CONFIG configured} via {@link StreamsConfig} or
-     * {@link KStream#through(String, Produced)}, or if the original {@link KTable}'s input
+     * {@link KStream#repartition(Repartitioned)}, or if the original {@link KTable}'s input
```

Comment:
Not sure if this update is necessary. This method is deprecated itself.

Reply:
Might as well make this update, since we may remove the methods at different times.